app/src/main/java/org/fossasia/pslab/communication/peripherals/MCP4728.java

```diff
@@ -64,9 +69,8 @@ public double setVoltage(String name, int v) {
         return setRawVoltage("name", v);
     }
 
-    public int getVoltage(String name) {
-        // todo : add method when resolved in pslab-python
-        return -1;
+    public Double getVoltage(String name) {
+        return this.values.get(name);
```

Comment:
Here the return type is double. In ScienceLab.java  where it is called 
``` 
public int getPV1() {
return this.dac.getVoltage("PV1"); 
}
```
the return type is int. Fix this.